warp10/src/main/java/io/warp10/script/processing/typography/PcreateFont.java

```diff
@@ -94,6 +96,12 @@ public Object apply(WarpScriptStack stack) throws WarpScriptException {
         //
         // FIXME(hbs): support WOFF/WOFF2 on top of TTF/OTF
         //
+        
+        // Encode the URL if it contains a whitespace
+        if (url.contains(" ")) {
+          url = URLEncoder.encode(url, StandardCharsets.UTF_8.name());
+        }
```

Comment:
This will fail because it will also escape part of the URL which should not be escaped resulting it "https%3A%2F%2F"...
You could unconditionally do a `url.replace(" ", "%20")`
or a nice solution from [SO](https://stackoverflow.com/q/10786042#answer-25735202):
```
      String urlStr = resolver.resolve(params.get(1).toString());

      InputStream in = null;
      
      try {
        URL url = new URL(urlStr);
        URI uri = new URI(url.getProtocol(), url.getUserInfo(), url.getHost(), url.getPort(), url.getPath(), url.getQuery(), url.getRef());
        
        in = new URL(uri.toASCIIString()).openStream();
```